.gitmodules

```diff
@@ -1,9 +1,6 @@
 [submodule "third_party/pybind11"]
 	path = tt_metal/third_party/pybind11
 	url = https://github.com/pybind/pybind11.git
-[submodule "third_party/lfs"]
```

Comment:
I guess this is an accidental change

Reply:
Hi Artem, this is for not for publish. Let me change it